zboxcore/sdk/sdk.go

```diff
@@ -832,27 +832,27 @@ func GetAllocationsForClient(clientID string) ([]*Allocation, error) {
 	return allocations, nil
 }
 
-func CreateAllocationWithBlobbers(datashards, parityshards int, size, expiry int64,
+func CreateAllocationWithBlobbers(name string, datashards, parityshards int, size, expiry int64,
 	readPrice, writePrice PriceRange, mcct time.Duration, lock int64, blobbers []string) (
 	string, error) {
 
 	return CreateAllocationForOwner(client.GetClientID(),
-		client.GetClientPublicKey(), datashards, parityshards,
+		client.GetClientPublicKey(), name, datashards, parityshards,
 		size, expiry, readPrice, writePrice, mcct, lock,
 		blobbers)
 }
 
-func CreateAllocation(datashards, parityshards int, size, expiry int64,
+func CreateAllocation(name string, datashards, parityshards int, size, expiry int64,
 	readPrice, writePrice PriceRange, mcct time.Duration, lock int64) (
 	string, error) {
 
 	return CreateAllocationForOwner(client.GetClientID(),
-		client.GetClientPublicKey(), datashards, parityshards,
+		client.GetClientPublicKey(), name, datashards, parityshards,
 		size, expiry, readPrice, writePrice, mcct, lock,
 		blockchain.GetPreferredBlobbers())
 }
 
-func CreateAllocationForOwner(owner, ownerpublickey string,
+func CreateAllocationForOwner(owner, ownerpublickey string, name string,
```

Comment:
Perhaps move the `name` param ahead to the first one to make it the same as CreateAllocation() function. Also, I saw the zboxcli is calling this function that pass the name as the first param